usaspending_api/references/urls.py

```diff
@@ -10,12 +10,13 @@
 
 
 urlpatterns = [
-    url(r'', include(guide_router.urls)),
     url(r'^locations/$', views.LocationEndpoint.as_view(mode_list)),
     url(r'^locations/geocomplete', views.LocationGeoCompleteEndpoint.as_view()),
     url(r'^agency/$', views.AgencyEndpoint.as_view(mode_list)),
     url(r'^agency/autocomplete', views.AgencyAutocomplete.as_view()),
     url(r'^recipients/autocomplete', views.RecipientAutocomplete.as_view()),
+    url(r'^guide/autocomplete', views.GuideAutocomplete.as_view()),
```

Comment:
Pinging @catherinedevlin, since she implemented the guide router when creating the original endpoint last sprint. I think taking this step makes a lot of sense, am not quite clear if/how we should continue to follow that pattern when adding, say, an `autocomplete`.